pkg/og/convert/pprof/profile.go

```diff
@@ -82,6 +82,14 @@ func (p *RawProfile) ParseToPprof(_ context.Context, md ingestion.Metadata) (res
 	return
 }
 
+func (p *RawProfile) isDotnetspy(md ingestion.Metadata) bool {
+	if md.SpyName == "dotnetspy" {
+		return true
+	}
+	stc := p.getSampleTypes()
+	return stc != nil && stc["inuse-space"] != nil
```

Comment:
slightly better dotnetspy detection, afaik "inuse-space" was only used in dotnet